File: core/src/main/java/network/hoz/grpcer/config/ClientMode.java
package network.hoz.grpcer.config;

public enum ClientMode {
    /**
     * Uses first available connection as main, any other is backup.
     */
    NORMAL,
    /**
     * Each call uses different connection.
     * If the connection fails, other is used.
     */
    ROUND_ROBIN
}

File: core/src/main/kotlin/network/hoz/grpcer/core/config/ClientMode.kt
package network.hoz.grpcer.core.config

enum class ClientMode {
    /**
     * Uses first available connection as main, any other is backup.
     */
    NORMAL,

    /**
     * Each call uses different connection.
     * If the connection fails, other is used.
     */
    ROUND_ROBIN
}